dump_utils.hpp

```diff
@@ -166,5 +166,34 @@ T readDBusProperty(sdbusplus::bus::bus& bus, const std::string& service,
     return retVal;
 }
 
+/**
+ *  @brief PEL Message severities for Dump related operations. Default is
+ * Informational
+ *
+ */
+enum class PelSeverity
+{
+    NOTICE,
+    INFORMATIONAL,
+    DEBUG,
+    WARNING,
+    CRITICAL,
+    EMERGENCY,
+    ALERT,
+    ERROR
+};
+
+/**
+ * @brief Create a new PEL message for dump Delete/Offload
+ *
+ * @param[in] additionalData - dump id, name and type.
+ * @param[in] sev - severity.
+ * @param[in] errIntf - D-Bus interface name.
+ * @return Returns void on success throws exception on error
+ **/
```

Comment:
void createPEL(
    sdbusplus::bus_t& bus,
    const std::unordered_map<std::string, std::string>& additionalData,
    const PelSeverity& sev, const std::string& errIntf);


caller has bus, you can pass that to this method

Reply:
<img width="1225" alt="Screenshot 2023-01-19 at 5 09 00 PM" src="https://user-images.githubusercontent.com/112170550/213433183-c376dcfe-3cb9-40af-8c06-1edd00ea84c1.png">
Getting compilation error. The sdbusplus wrapper has no direct access to it's internal sd_bus* object